test/pthread/main.c

```diff
@@ -84,6 +84,67 @@ static int test_mutex_with_concurrent_counter(void) {
     return 0;
 }
 
+// ============================================================================
+// The test case of robust mutex
+// ============================================================================
+
+struct thread_robust_arg {
+    int                         ti;
+    volatile int                *global_count;
+    pthread_mutex_t             *mutex;
+};
+
+static void *thread_worker(void *_arg) {
+    struct thread_robust_arg *arg = _arg;
+    int err = pthread_mutex_lock(arg->mutex);
+    if (err == EOWNERDEAD) {
```

Comment:
It would be good to check for the errors other than `EOWNERDEAD` to rule out any unexpected errors.

Reply:
Done